packages/client/src/views/Settings/SettingsPage.tsx

```diff
@@ -312,120 +289,103 @@ class SettingsView extends React.Component<IProps, IState> {
+          disabled: true
+        }
+      },
+      {
+        label: intl.formatMessage(messages.profileImage),
+        value: <Avatar avatar={userDetails?.avatar} name={englishName} />,
+        action: {
+          id: 'BtnChangeAvatar',
+          label: intl.formatMessage(buttonMessages.change),
+          handler: () => this.fileUploaderRef.current?.click()
+        }
       }
     ]
+
     return (
       <>
         <Header title={intl.formatMessage(messages.settingsTitle)} />
         <Navigation />
         <BodyContainer>
-          <Container>
-            <SettingsTitle>
-              {intl.formatMessage(messages.settingsTitle)}
-            </SettingsTitle>
-            <Content>
-              <Left>
-                {sections.map((sec, index: number) => (
-                  <ListView key={index} {...sec} />
-                ))}
-                <Version>
-                  <span>OpenCRVS v1.1.0</span>
-                  <span>{process.env.REACT_APP_VERSION || 'development'}</span>
-                </Version>
-              </Left>
-              <Right>
-                <ImageLoader
-                  onImageLoaded={this.handleImageLoaded}
-                  onLoadingStarted={this.toggleAvatarChangeModal}
-                  onError={(imageLoadingError) =>
-                    this.setState({ imageLoadingError })
-                  }
-                >
-                  <Avatar
-                    className="tablet clickable"
-                    avatar={userDetails?.avatar}
-                    name={englishName}
+          <Content title={intl.formatMessage(messages.settingsTitle)}>
+            <ListViewSimplified>
+              {items.map((item) => {
+                return (
+                  <ListViewItemSimplified
+                    key={item.label}
+                    label={<LabelContainer>{item.label}</LabelContainer>}
+                    value={<ValueContainer>{item.value}</ValueContainer>}
+                    actions={[
+                      <LinkButton
+                        id={item.action.id}
+                        onClick={item.action.handler}
+                        disabled={item.action.disabled}
+                      >
+                        {item.action.label}
+                      </LinkButton>
+                    ]}
                   />
-                  <AvatarLarge
-                    className="desktop clickable"
-                    avatar={userDetails?.avatar}
-                    name={englishName}
-                  />
-                </ImageLoader>
-              </Right>
-            </Content>
-          </Container>
+                )
+              })}
+            </ListViewSimplified>
+            <HiddenInput
+              ref={this.fileUploaderRef}
+              id="image_file_uploader_field"
+              type="file"
+              accept={ALLOWED_IMAGE_TYPE.join(',')}
+              onChange={this.handleSelectFile}
+            />
```

Comment:
Maybe we can reuse `ImageLoader`

Reply:
Yeah, We can . I did that. Thank you for your suggestion.